### PR TITLE
feat: Google Ads API response to Cassandra format

### PR DESCRIPTION
**Pull request by shubhank-v**
### What changes are proposed in this PR?
---
* This PR converts the google ads API response to Cassandra's expected format

### How was this tested?
---
*  Unit test.

### CODE REVIEW

File: hip_data_tools/common.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
                          "most_specific_location"]
 
 LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
```
Comment (samithdisal) [inline hip_data_tools/common.py:21]: Isn't this by default points to INFO and should be configured by an Env flag?

File: hip_data_tools/google/googleads.py [hunk 2]
```diff
@@ -580,13 +580,27 @@ def upload_conversions(self, data):
...
-        partial_failure = getattr(result, "partial_failure_error", None)
-        uploaded = getattr(result, "results", None)
-        return uploaded, partial_failure
+        return self._get_response(data, result)
 
     def _upload_mutations_batch(self, mutations):
         return self._get_service().upload_click_conversions(mutations)
 
+    def _get_conversions(self, data):
+        conversions = getattr(data, "conversions")[0]
```
Comment (samithdisal) [inline hip_data_tools/google/googleads.py:589]: Do we need to handle the scenario where the concersions contain 0 elements?

Comment (shubhank-v) [reply hip_data_tools/google/googleads.py:589]: https://developers.google.com/google-ads/api/reference/rpc/v10/UploadClickConversionsRequest
conversions will be a list.

File: hip_data_tools/common.py (hunk 1 above)
Comment (shubhank-v) [reply hip_data_tools/common.py:21]: Good idea, toggling via env.

File: hip_data_tools/google/googleads.py (hunk 2 above)
Comment (samithdisal) [reply hip_data_tools/google/googleads.py:589]: My concern was if we want to check for `IndexError` and inform that there are no conversions. But it seems it is not the case. So I'll approve as it is.

**Review by samithdisal — APPROVED** [review]

**Pull request merged**